### PR TITLE
Add support to deploy a pipeline's airflow-spec to the Airflow environment

### PR DESCRIPTION
**Pull request by rajivpb**

### CODE REVIEW

File: google/datalab/contrib/bigquery/commands/_bigquery.py [hunk 1]
```diff
@@ -24,6 +24,10 @@ def _create_pipeline_subparser(parser):
                                help='The name of the Composer environment.')
   pipeline_parser.add_argument('-l', '--location', type=str,
                                help='The name of the Cloud location for the environment.')
+  pipeline_parser.add_argument('-d', '--gcs_dag_bucket', type=str,
+                               help='The GCS bucket for the dags for Airflow.')
```
Comment (chmeyers) [inline google/datalab/contrib/bigquery/commands/_bigquery.py]: GCS -> "Google Cloud Storage" for branding in the help.  flag name is fine.

File: google/datalab/contrib/pipeline/airflow/_airflow.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  Cloud Airflow service.
+
+  This object can be used to generate the python airflow spec.
+  """
+
+  def __init__(self, gcs_dag_bucket, gcs_dag_file_path=None):
+    """ Initializes an instance of a Airflow object.
+
+    Args:
+      gcs_dag_bucket: Zone in which Airflow environment has been created.
```
Comment (chmeyers) [inline google/datalab/contrib/pipeline/airflow/_airflow.py]: Zone-> Bucket?

File: tests/bigquery/pipeline_tests.py [hunk 3]
```diff
@@ -472,7 +472,8 @@ def test_pipeline_cell_golden(self, mock_client_get_bucket, mock_client, mock_bl
 
     mock_environment.return_value = env
     args = {'name': 'bq_pipeline_test', 'environment': 'foo_environment',
-            'location': 'foo_locaiton'}
+            'location': 'foo_locaiton', 'gcs_dag_bucket': 'foo_bucket',
```
Comment (chmeyers) [inline tests/bigquery/pipeline_tests.py]: locaiton -> location?

File: google/datalab/contrib/bigquery/commands/_bigquery.py (hunk 1 above)
Comment (rajivpb) [reply google/datalab/contrib/bigquery/commands/_bigquery.py]: Done.

File: google/datalab/contrib/pipeline/airflow/_airflow.py (hunk 2 above)
Comment (rajivpb) [reply google/datalab/contrib/pipeline/airflow/_airflow.py]: Done.

File: tests/bigquery/pipeline_tests.py (hunk 3 above)
Comment (rajivpb) [reply tests/bigquery/pipeline_tests.py]: Done.

**Pull request merged**